### PR TITLE
Add required check that all tags are on master branches

### PR DESCRIPTION
**Pull request by jbgi**

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -36,6 +36,9 @@ let
     checks = recurseIntoAttrs {
       # `checks.tests` collect results of executing the tests:
       tests = collectChecks haskellPackages;
+
+      # checks that every tags in cabal.project exists on master branches:
```
Comment (mrBliss) [inline default.nix]: Using "tags" confused me initially. This is about commit hashes, not about git tags, but in `cabal.project`, `tag` is used for that purpose.

btw, "every tag" or "all tags .. exist"

**Comment by jbgi** [comment]
Example output for when the check is failing:
https://buildkite.com/input-output-hk/ouroboros-network/builds/6561#060612c9-fed2-4e36-a01c-dd35e654904b
(used an iohk-monitoring commit not on master yet)

**Comment by mrBliss** [comment]
@jbgi This PR got out of date. Can you update it and merge it?

**Comment by mrBliss** [comment]
> @jbgi This PR got out of date. Can you update it and merge it?

Rebased it and it still works!

Better late than never :grimacing:

Thanks again @jbgi!

**Comment by mrBliss** [comment]
bors merge

**Pull request merged**